codacy/values-production.yaml

```diff
@@ -197,54 +219,68 @@ core:
       memory: 1500Mi
     requests:
       cpu: 100m
-      memory: 750Mi
-
+      memory: 200Mi
 engine:
   replicaCount: 2
   resources:
     limits:
       cpu: 2000m
       memory: 15000Mi
     requests:
-      cpu: 1000m
-      memory: 10000Mi
-
+      cpu: 200m
+      memory: 500Mi
 worker-manager:
   replicaCount: 2
   resources:
     limits:
       cpu: 500m
       memory: 1000Mi
     requests:
-      cpu: 100m
-      memory: 500Mi
+      cpu: 200m
+      memory: 200Mi
   config:
     workers:
-      genericMax: 10
```

Comment:
is this deliberate?

Reply:
Yes, I think we realized that 10 is clearly too much. Also, 2 is the default for doplicated. We better go with that.